pkg/network/multus/nad.go

```diff
@@ -42,3 +50,26 @@ func NetAttachDefNamespacedName(namespace, fullNetworkName string) types.Namespa
 		Name:      fullNetworkName,
 	}
 }
+
+func GetNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (networkToResourceMap map[string]string, err error) {
```

Comment:
It will be nice to drop the `Get` prefix and `Map` tail. It is not a getter and we do not like to embed in the names the type. Can be done as part of the refactor in the following commit.

Reply:
Done.